### PR TITLE
merging purpleair csv from a folder and muching columns step 1

### PR DESCRIPTION
**Pull request by collaer**
A very first step of muching purple air raw csv from a folder ./sample folder

- It iterate csv files (filtering only Principals)
- It grab information from filenames (x, y and station id)
- It remove useless columns and keep/fill/rename correct ones
- datetime,pm25,type,sensor,station_id,x,y

Append for each file into a new one name merged.csv

Need to be done for all purpleair files (select all the ground stations we want and reconfigure eventually to something else that a folder)

For the rest of the team that give us a first merged.csv to work on (appending new columns / features)

Please review and merge we need this early step to finalize other data source muching logics

### CODE REVIEW

File: data/purpleair/merge-pa.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    for file in os.listdir(folder):
+        logging.info('Hiting file {}'.format(file))
+        if file.endswith(".csv") and ('Primary' in file):
+            logging.info('Adding file {} to list'.format(file))
+            filesList += [file]
+    return filesList
+
+def getFileData(file):
+    df = pd.read_csv(file)
+    columns = list(df.columns)
```
Comment (collaer) [inline data/purpleair/merge-pa.py]: I ve just dowload the first 2 stations data to build this example and their datamodel was already different (not sames columns, etc) so I add all ccolumns and try to keep the 2 we are interested in (pm25 y datatime)
But this should probably be adapted to cover the rest of the data if they change the pm25 column name or the date time one or the content date formatting.

**Comment by collaer** [comment]
As soon as you review/merge this PR I can continue to work so please post a slack directly after so I can see it.

File: data/purpleair/merge-pa.py (hunk 1 above)
Comment (ayeshabaig) [reply data/purpleair/merge-pa.py]: Are we planning on leaving the CSV locally?

**Comment by ayeshabaig** [comment]
Looks good for the first wrangling iteration.

**Pull request closed without merging**